### PR TITLE
test:Preferences in the Menu Bar opens the Settings page

### PR DESCRIPTION
**Pull request by codedsun**
#### Summary

e2e test:Preferences in the Menu Bar opens the Settings page

#### Ticket Link

  Fixes https://github.com/mattermost/desktop/issues/1893

#### Checklist

- [ ] Added or updated unit tests (required for all new features)
- [ ] Has UI changes
- [ ] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
- [ ] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
- [ ] executed `npm run lint:js` for proper code formatting

#### Device Information
This PR was tested on: 

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by codedsun** [comment]
@devinbinnie One question, how do i close the settings window which is opened by shortcut. I tried `settingsWindow.close()` but it throws error 
```
Timeout of 30000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves
```

**Comment by codedsun** [comment]
@devinbinnie @M-ZubairAhmed The doc also says that closing the window can be done by `win.close()` https://www.electronjs.org/docs/v14-x-y/api/browser-window#winclose

**Comment by codedsun** [comment]
@devinbinnie @M-ZubairAhmed I just resolved using the shortcuts to close the window. Please review. Thanks

File: e2e/specs/menu_bar/file_menu.test.js [hunk 1]
```diff
@@ -39,4 +39,28 @@ describe('file_menu/dropdown', function desc() {
             settingsWindow.should.not.be.null;
         }
     });
+
+    it('MM-T804 Preferences in Menu Bar open the Settings page', async () => {
+        const mainWindow = this.app.windows().find((window) => window.url().includes('index'));
+        mainWindow.should.not.be.null;
+        if (process.platform === 'win32' || process.platform === 'linux') {
```
Comment (devinbinnie) [inline e2e/specs/menu_bar/file_menu.test.js:46]: Can we move this check to the outside of the test and just check that `process.platform !== 'darwin'`?

**Review by devinbinnie — CHANGES_REQUESTED** [review]
Thanks for the PR @codedsun! Just one comment to resolve :)

File: e2e/specs/menu_bar/file_menu.test.js (hunk 1 above)
Comment (codedsun) [reply e2e/specs/menu_bar/file_menu.test.js]: ✅ Done as requested! Thanks

**Review by devinbinnie — APPROVED** [review]
LGTM! Thanks @codedsun!

**Comment by codedsun** [comment]
@M-ZubairAhmed Please review when time allows. Thanks

**Review by M-ZubairAhmed — APPROVED** [review]
🆗  It looks good to me. Thank you

**Comment by devinbinnie** [comment]
@codedsun Can you resolve conflicts? Then I can merge this :)

**Comment by codedsun** [comment]
@devinbinnie Yes i will do it tomorrow. Thanks

**Comment by codedsun** [comment]
@devinbinnie Done the changes..

**Pull request merged**